### PR TITLE
[PDX203] mixer_paths: Add modem audio proxy ports, set DPI for 4k resolution

### PR DESCRIPTION
**Pull request by kholk**
The proxy ports are used to get call recording and call audio
proxying to displayport.

PLEASE PORT THIS TO PDX206!!!!

This patchset also changes the DPI for the new resolution, refer to:
https://github.com/sonyxperiadev/kernel/pull/2400

### CODE REVIEW

File: rootdir/vendor/etc/mixer_paths.xml [hunk 1]
```diff
@@ -261,6 +261,11 @@
     <ctl name="DISPLAY_PORT_RX_Voice Mixer VoiceMMode1" value="0" />
     <!-- Miltimode Voice1 end-->
 
+    <!-- Multimode Voice1 proxy-Port -->
+    <ctl name="PROXY_RX_Voice Mixer VoiceMMode1" value="0" />
+    <ctl name="VoiceMMode1_Tx Mixer PROXY_TX_MMode1" value="0" />
+    <!-- Miltimode Voice1 end-->
```
Comment (MarijnS95) [inline rootdir/vendor/etc/mixer_paths.xml:267]: Multi*

**Review by MarijnS95 — COMMENTED** [review]
Similar commit on CAF: https://source.codeaurora.org/quic/la/platform/hardware/qcom/audio/commit/?id=61c3215dd950b795b890c539e7582e75a82aab43

Curious that we didn't get call screening on stock, are you sure this has much to do with displayport though?

By the way, shouldn't we turn on all the extra booleans/flags to make it usable, besides just enabling the paths?

File: rootdir/vendor/etc/mixer_paths.xml (hunk 1 above)
Comment (MarijnS95) [reply rootdir/vendor/etc/mixer_paths.xml:267]: EDIT: That CAF commit (EDIT2: and all other devices) is where you got the typo from :joy: 

```diff
     <ctl name="VoiceMMode1_Tx Mixer AFE_PCM_TX_MMode1" value="0" />
+    <!-- Multimode Voice1 proxy-Port -->
+    <ctl name="PROXY_RX_Voice Mixer VoiceMMode1" value="0" />
+    <ctl name="VoiceMMode1_Tx Mixer PROXY_TX_MMode1" value="0" />
     <!-- Miltimode Voice1 end-->
```

EDIT3: No, you copied it from line 262 right above.

Comment (kholk) [reply rootdir/vendor/etc/mixer_paths.xml:267]: I've copied that from the kona configs that I've found in the audio hal....

**Pull request merged**